index.html

```diff
@@ -41,13 +41,22 @@ <h1 id="annoying-message">Annoyance loading...</h1>
             </div>
         </div>
 
-        <script   src="https://code.jquery.com/jquery-2.2.3.min.js" integrity="sha256-a23g1Nt4dtEYOj7bR+vTu7+T8VP13humZFBJNIYoEJo=" crossorigin="anonymous"></script>
         <script type="text/javascript" src="./annoyingmicah.js"></script>
 
         <script type="text/javascript">
-            (function($) {
-                $('#annoying-message').html(annoyingMicahFilter());
-            })(jQuery);
+            // Stole (and modified) from http://stackoverflow.com/a/901144
```

Comment:
Really concerned about this line, we do NOT want people to know that there's third party code in this application, it's already been sold to the customer as secure